src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,83 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+
+type Props = {
+  currentTodo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({ currentTodo }) => {
+  const [userInfo, setUserInfo] = useState<User>();
+  const [isLoader, setIsLoader] = useState(false);
+  const {
+    userId,
+    id,
+    title,
+    completed,
+  } = currentTodo;
+
+  const loadDataUser = async (currentUserId: number) => {
+    try {
+      setIsLoader(true);
+      const data = await getUser(currentUserId);
+
+      setUserInfo(data);
+    } catch (error) {
+      window.console.log('error = ', error);
```

Comment:
Instead of console.log, you should handle the error properly. You can show it in UI.